jobs-shell/script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_jobs_shell_script]
```

Comment:
The region tags will be cloudrun_jobs_exit_process and cloudrun_jobs_env_vars once the sample is update to mimic https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/run/jobs/index.js